springdoc-openapi-webmvc-core/src/main/java/org/springdoc/webmvc/api/OpenApiResource.java

```diff
@@ -265,6 +262,12 @@ protected void calculatePath(Map<String, Object> restControllers, Map<RequestMap
 		}
 	}
 
+	private Comparator<Map.Entry<RequestMappingInfo, HandlerMethod>> byReversedRequestMappingInfos() {
```

Comment:
Wasn't sure, if you want to share this somehow between webflux and webmvc?!

Reply:
It would have been an ideal to refactor it in the AbstractOpenApiResource. 
But, it doesn't seem to be simple to achieve.
Your solution looks fine!